packages/frontend/src/lib/conversation/ToolCallMessage.spec.ts

```diff
@@ -0,0 +1,127 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { expect, test, beforeEach, describe } from 'vitest';
+import { fireEvent, render, within } from '@testing-library/svelte';
+import ToolCallMessage from '/@/lib/conversation/ToolCallMessage.svelte';
+
+describe('ToolCallMessage component', () => {
+  let container: HTMLElement;
+  describe('with result', () => {
+    beforeEach(() => {
+      container = render(ToolCallMessage, {
```

Comment:
remark: Here you don't need it because you don't mock anything, but it is always preferable to have `vi.resetAllMock()` to avoid forgetting when we need it.

Reply:
I will add it.
However, as I stated in the previous PR, striving to not use mocks usually leads to a healthier code-base ;)

Reply:
> However, as I stated in the previous PR, striving to not use mocks usually leads to a healthier code-base ;)

That's an interesting opinion